Sources/Testing/Traits/Tags/Tag+Predefined.swift

```diff
@@ -0,0 +1,104 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+// MARK: Primary colors
+
+extension Tag {
+  /// A tag representing the color red.
+  public static var red: Self {
+    Tag(kind: .staticMember("red"))
+  }
+
+  /// A tag representing the color orange.
+  public static var orange: Self {
+    Tag(kind: .staticMember("orange"))
+  }
+
+  /// A tag representing the color yellow.
+  public static var yellow: Self {
+    Tag(kind: .staticMember("yellow"))
+  }
+
+  /// A tag representing the color green.
+  public static var green: Self {
+    Tag(kind: .staticMember("green"))
+  }
+
+  /// A tag representing the color blue.
+  public static var blue: Self {
+    Tag(kind: .staticMember("blue"))
+  }
+
+  /// A tag representing the color purple.
+  public static var purple: Self {
+    Tag(kind: .staticMember("purple"))
+  }
+
+  /// Whether or not this tag represents a color predefined by the testing
+  /// library.
+  ///
+  /// Predefined color tags are any of these values:
+  ///
+  /// - ``Tag/red``
+  /// - ``Tag/orange``
+  /// - ``Tag/yellow``
+  /// - ``Tag/green``
+  /// - ``Tag/blue``
+  /// - ``Tag/purple``
+  public var isPredefinedColor: Bool {
```

Comment:
This property probably does not need to be `public` but I'm leaving it as such for now since it already is declared.

Reply:
I think I'd prefer to see the visibility reduced on this one.

Reply:
Oh alright.